ui/component/claimListDiscover/index.js

```diff
@@ -13,6 +13,7 @@ import { selectFollowedTags } from 'redux/selectors/tags';
 import { selectMutedAndBlockedChannelIds } from 'redux/selectors/blocked';
 import { doFetchOdyseeMembershipForChannelIds } from 'redux/actions/memberships';
 import { selectClientSetting, selectShowMatureContent, selectLanguage } from 'redux/selectors/settings';
```

Comment:
When I reset
- [ ] Portals come in between
- [ ] "Customize" is missing

<img width="721" alt="image" src="https://github.com/OdyseeTeam/odysee-frontend/assets/64950861/f0a3daf6-99e7-422f-9de7-2c0005e8f250">

Reply:
In this sequence, the cog seems missing.  Doesn't really matter since still accessible from the sidebar, I guess

<img width="221" alt="image" src="https://github.com/OdyseeTeam/odysee-frontend/assets/64950861/77dc8134-e6d8-402e-9a04-0b37f1486810">